models/User.js

```diff
@@ -14,6 +14,14 @@ const UserSchema = new Schema({
     type: String,
     required: true
   },
+  courseIDs: {
+    type: [String],
+    required: false
+  },
+  chatIDs: {
+    type: [String],
+    required: false
+  },
```

Comment:
Is there a need for having them ordered? If not, you can just use an object. It would be faster to update/delete, specific elements.